tests/suites/1_stateful/10_iceberg/10_0002_iceberg_engine.result

```diff
@@ -0,0 +1,17 @@
+>>>> drop table if exists test_iceberg;
+>>>> create table test_iceberg engine = iceberg 'fs://${ROOT}/';
+>>>> select * from test_iceberg order by id, data;
+1	a
+2	b
+3	c
+4	d
+5	e
+6	d
+<<<<
+>>>> show create table test_iceberg;
+test_iceberg	CREATE TABLE `test_iceberg` (
+  `id` INT NOT NULL,
+  `data` VARCHAR NOT NULL
+) ENGINE=ICEBERG
```

Comment:
seems  the location is missing in show stmt

Reply:
I think there is the same for `create table with external location`:

```
pub struct ShowCreateTablePlan {
    /// The catalog name
    pub catalog: String,
    /// The database name
    pub database: String,
    /// The table name
    pub table: String,
    /// The table schema
    pub schema: DataSchemaRef,
}
```

----

the problem is    `<url> [connection=]`  is translated  to `stroage parameters`.
need a way to restore it? maybe not always feasible.

Reply:
> the problem is `<url> [connection=]` is translated to `stroage parameters`.
> need a way to restore it?

Yep, we are not ready for transform stroage parameters into url yet.

Reply:
> seems the location is missing in show stmt

fixed